packages/app/src/Element/NoteFooter.tsx

```diff
@@ -102,15 +118,39 @@ export default function NoteFooter(props: NoteFooterProps) {
     }
   }
 
+  async function zapClick(e: React.MouseEvent) {
+    if (zapping || e.isPropagationStopped()) return;
+
+    const lnurl = author?.lud16 || author?.lud06;
+    if (webln?.enabled && lnurl) {
+      setZapping(true);
+      try {
+        const handler = new LNURL(lnurl);
+        await handler.load();
+
+        const zap = await publisher.zap(prefs.defaultZapAmount * 1000, ev.PubKey, ev.Id);
```

Comment:
Need to check `handler.canZap` should we allow Non-zap with fast button?

Reply:
I think falling back to non-zap if not supported is fine, still want the instant UX imo.